File: DropZone/DropZone.Web/src/models/aads/aadType.model.ts
import { ManufacturerModel } from "../manufacturers/manufacturer.model";

export class AADTypeModel {
	public Id: number;
	public Type: string;
	public Height: number;
	public Speed: number;
    public ModifiedAt: Date;

	public ManufacturerId: number;
    
	public Manufacturer: ManufacturerModel;
};

File: DropZone/DropZone.Core/Models/AADType/AADTypeModel.cs
using System;

namespace DropZone.Core.Models
{
    public class AADTypeModel
    {
        public long Id { get; set; }

        public string Type { get; set; }

        public double Height { get; set; }

        public double Speed { get; set; }

        public DateTime ModifiedAt { get; set; }


        public long ManufacturerId { get; set; }


        public ManufacturerModel Manufacturer { get; set; }
    }
}
